local/tests/run_all_tests

```diff
@@ -17,4 +17,4 @@ $SCRIPT_DIR/run_untrusted_test minimize_task
 $SCRIPT_DIR/run_utask_test analyze_task
 
 echo 'Running end-to-end integration tests...'
-python butler.py integration_tests
+python butler.py integration_tests
```

Comment:
Please don't make unnecessary changes.

Reply:
This is an empty diff, not really a change

Reply:
This pollutes the log and the blame and is a nuisance for reviewers. Can you remove it please?